common/components/controllers/VideoController.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import React from "react";
+
+// Shows embedded video
+class VideoController extends React.PureComponent {
+  constructor(): void {
+    super();
+  }
+
+  render(): React$Node {
+    return (
+      <iframe
```

Comment:
I'd suggest wrapping this iframe in a <div> so we can do some positioning. Assuming the div is named something like youtube-video-container, then:

```css
.youtube-video-container {
  position: relative;
  overflow: hidden;
  width: 95%;
}

.youtube-video-container::after {
  display: block;
  content: "";
  padding-top: 56.25%;
}

.youtube-video-container iframe {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
}
```

I just searched 'responsive youtube embed' or something similar and came up with [this article](https://techstacker.com/how-to-responsive-youtube-videos/) but I'm sure there are others. My goal was to center it, enlarge it, and then push it off the footer a bit because it's really big.

It's not perfect but I think it's an improvement. Try it out and see if you like it?